roofit/histfactory/src/Measurement.cxx

```diff
@@ -268,8 +268,16 @@ void RooStats::HistFactory::Measurement::PrintXML( std::string directory, std::s
   // Then, create xml files for each channel
 
   // First, check that the directory exists:
-
-  if( directory != "" && gSystem->OpenDirectory( directory.c_str() ) == 0 ) {
+  auto testExists = [](const std::string& directory) {
+    void* dir = gSystem->OpenDirectory(directory.c_str());
+    bool exists = dir != 0;
```

Comment:
Maybe using nullptr is more appropriate than the "traditional" 0?

Reply:
Done